packages/utils/index.ts

```diff
@@ -1,2 +1,2 @@
 export * from './src'
-export { default } from './src'
+export { options as default  } from './src'
```

Comment:
why?

Reply:
Noted - I'm not sure we want to export options as the default from utils